neo4j-jdbc-bolt/src/main/java/org/neo4j/jdbc/bolt/BoltNeo4jConnection.java

```diff
@@ -43,7 +43,7 @@ public interface BoltNeo4jConnection extends Connection {
 	 *
 	 * @return the internal session
 	 */
-	Session getSession();
+	Session getOrCreateSession();
```

Comment:
alternatively, we could keep the session open for the whole connection lifetime, instead of closing it at commit.
I think that alternative would be better.